### PR TITLE
sweepbatcher: set max batch size to 1000 sweeps

### PR DESCRIPTION
**Pull request by starius**
This is needed to avoid non-standard batch transactions (larger than 400k wu). A non-cooperative input is 393 wu, so 1000 inputs are still under 400k wu.

#### Pull Request Checklist
- [ ] Update `release_notes.md` if your PR contains major features, breaking changes or bugfixes

### CODE REVIEW

File: sweepbatcher/sweep_batcher_test.go [hunk 1]
```diff
@@ -1176,6 +1176,156 @@ func testDelays(t *testing.T, store testStore, batcherStore testBatcherStore) {
...
+	var runErr error
+	go func() {
+		defer wg.Done()
+		runErr = batcher.Run(ctx)
+	}()
+
+	// Wait for the batcher to be initialized.
+	<-batcher.initDone
+
+	const swapsNum = 2500
```
Comment (bhandras) [inline sweepbatcher/sweep_batcher_test.go:1227]: To reduce test runtime, would just 1001 work? Also the test seems to be logging excessively, so perhaps we could reduce that too. wdyt?

Comment (starius) [reply sweepbatcher/sweep_batcher_test.go]: Great idea! I changed the number of swaps to 1001. Disabling logging has negligible effect.

Comment (bhandras) [reply sweepbatcher/sweep_batcher_test.go]: Disabling logging is helpful when running `make unit` locally, as it's easier to follow test logs.

**Review by bhandras — APPROVED** [review]

File: sweepbatcher/sweep_batcher_test.go (hunk 1 above)
Comment (starius) [reply sweepbatcher/sweep_batcher_test.go]: Disabled logging in the test.

File: sweepbatcher/sweep_batch.go [hunk 2]
```diff
@@ -462,6 +467,13 @@ func (b *batch) addSweep(ctx context.Context, sweep *sweep) (bool, error) {
 		return true, nil
 	}
 
+	// Enforce MaxSweepsPerBatch. If there are already too many sweeps in
+	// the batch, do not add another sweep to prevent the tx from becoming
+	// non-standard.
+	if len(b.sweeps) >= MaxSweepsPerBatch {
+		return false, nil
```
Comment (hieblmi) [inline sweepbatcher/sweep_batch.go:474]: does it make sense to log such an occurence?

**Review by hieblmi — APPROVED** [review]
LGTM 💾

File: sweepbatcher/sweep_batch.go (hunk 2 above)
Comment (starius) [reply sweepbatcher/sweep_batch.go:479]: I added Info log message in cases when addSweep returns accept=false and err=nil in a separate commit "sweepbatcher: log the reason of acceptance failure".

**Pull request merged**